index.Rmd

```diff
@@ -0,0 +1,360 @@
+
+**Top Five most rated Books on Goodreads:**
+
+![r gr_top_ten[1]](https://images.gr-assets.com/books/1447303603m/2767052.jpg){width="125"} ![r gr_top_ten[2]](https://images.gr-assets.com/books/1474154022m/3.jpg){width="124"} ![r gr_top_ten[3]](https://images.gr-assets.com/books/1361039443m/41865.jpg){width="124"} ![r gr_top_ten[4]](https://images.gr-assets.com/books/1361975680m/2657.jpg){width="120"} ![r gr_top_ten[5]](https://images.gr-assets.com/books/1490528560m/4671.jpg){width="119"}
+
+This data is certainly helpful for me as I have read all five of these books and they do fall squarely into my preferred genre, however I need to understand the customer more deeply to know which direction to go.
+
+### Preliminary Distributions
+
+```{r echo=FALSE, fig.show="hold", message=FALSE, warning=FALSE, out.width="30%"}
+
+ga_clean %>% 
+  ggplot(aes(x = average_rating)) +
+  geom_histogram(fill = lgr_pink, color = "grey" ) +
+  labs(title = "Average Rating on Goodreads",
+       x = "rating",
+       y = "") +
+  coord_cartesian(xlim = c(3,5)) +
+  theme_classic() +
+  theme(axis.text = element_text(size = 12),
+        title = element_text(size = 15),
+        plot.title = element_text(face = "bold"))
+
+ga_clean %>% 
+  ggplot(aes(x = pages)) +
+  geom_histogram(fill = lgr_pink, color = "grey" ) +
+  labs(title = "Book Length",
+       x = "pages",
+       y = "") +
+  coord_cartesian(xlim = c(0,900)) +
+  theme_classic() +
+  theme(axis.text = element_text(size = 12),
+        title = element_text(size = 15),
+        plot.title = element_text(face = "bold"))
+
+ga_clean %>% 
+  filter(publication_year > 1600) %>% 
+  ggplot(aes(x = publication_year)) +
+  geom_histogram(fill = lgr_pink, color = "grey" ) +
+  labs(title = "Distribution of Publication Year",
+       x = "Year",
+       y = "") +
+  scale_x_continuous(limits = c(1900,2025)) +
+  theme_classic() +
+  theme(axis.text = element_text(size = 12),
+        title = element_text(size = 15),
+        plot.title = element_text(face = "bold"))
+  
+
+```
+
+Looking over this information I can get a preliminary picture of the average reader. The average book length is `r format(mean(ga_clean$pages,na.rm =T),digits = 3)` pages, with 4 being the median average rating. this confirms my assumption that people are more likely to rate books they like and skew their rating more generously than yelp reviewers.additionally I can see that the majority of books read (or at least rated on Goodreads) were published in this century. This is not surprising as Goodreads first launched in 2007. A fair assumptions to make here, based on my own habits, is that users are generally tracking, rating, and reviewing the books they are currently and want to read rather than going back to add in all the books read during the 1997 summer reading challenge.
+
+### Bookshelf Breakdown
+
+```{r, include=FALSE}
+tag_clean <- 
```

Comment:
This is a really nice bit of code. Nice job handling the list column via `unnest` combined with some regex and the use of `across`. Very nice.